### ISSUE TITLE
Author Archive color contrast failure

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/student-loan-planner-theme#332]
https://studentloanstg.wpengine.com/author/Travis-Hornsby/#main-content

On focus this gets purple text which fails contrast:
<img width="333" alt="Screenshot 2023-10-15 at 12 14 03 PM" src="https://github.com/equalizedigital/student-loan-planner-theme/assets/13153456/71659711-39ba-4561-ab3e-d110804bcc2d">

Keep the text black.
